### ISSUE TITLE
Compatibility issue with Django 3.0

### ISSUE THREAD

**Issue by thibault** [rbarrois/django_xworkflows#31]
Django 3.0 was released this month.

When we tried to upload our project to django 3.0, we realized `django-xworflows` is not compatible.

Here is the problem we met:

```
python manage.py runserver
…
File "/home/aides/django_xworkflows/django_xworkflows/models.py", line 19, in <module>
    from django.utils.encoding import force_text, python_2_unicode_compatible
ImportError: cannot import name 'python_2_unicode_compatible' from 'django.utils.encoding' (/home/aides/.virtualenvs/aides/lib/python3.7/site-packages/django/utils/encoding.py)
```

The issue is caused by [the following removal of aliases from the django codebase](https://docs.djangoproject.com/en/3.0/releases/3.0/#removed-private-python-2-compatibility-apis).

**Comment by kemar** [comment]
Hi @rbarrois :) I'm facing the same issue. Do you want to keep [Python 2 compatibility](https://www.python.org/dev/peps/pep-0373/#maintenance-releases)? I could make a PR using `six.python_2_unicode_compatible()` in this case?

**Comment by rbarrois** [comment]
Fixed in 6a5bd581b839c5263f98ab522ebd42f406fb2fc0, thanks for the contributions!
